### ISSUE TITLE
Bluescreen 1.9.22

### ISSUE THREAD

**Issue by githubbapoopa** [clsid2/mpc-hc#1789]
I got bluescreens when fast forward seeking or playing at >= 4x speed. This only happens on 1.9.22. while 1.9.21.2 is fine.
I'm using the portable versions and changed nothing else in the system. This error is hard to find I think because of bluescreens. mpc-hc has lost its .ini file after the bluescreen also.

Win10+ NVMe SSD

What can I do to assist you with this bug?

**Comment by clsid2** [comment]
Bluescreens are caused by a driver or hardware issue. It is not because of a fault in the player that I can fix.

This tool can often give more details about a possible cause:
https://www.nirsoft.net/utils/blue_screen_view.html

**Comment by githubbapoopa** [comment]
Well normally I would agree. But as I said I changed nothing in my rock solid system expect 1.9.21.2 with 1.9.22. I'm using mpc-hc for quite some time and never had that kind of behaviour before. mpc-hc always had been stable. And I check for new updates frequently.
1.9.22 seems to introduce that bug. Only occurs while seeking and playing at >= 4x speed (that seems the most prone error).
For me it seems obvious that the error was introduced with any of the new features in 1.9.22 (there has been many).

Reverting to 1.9.21.2 and checking 1.9.22 when I got the time as blueescreens are really nasty in a productive envvironement.
